component/client-handler/src/main/java/org/wso2/carbon/identity/oauth2/token/handler/clientauth/jwt/validator/JWTValidator.java

```diff
@@ -228,12 +228,10 @@ private boolean validateIssuer(String issuer, String consumerKey) throws OAuthCl
 
     // "The Audience SHOULD be the URL of the Authorization Server's Token Endpoint", if a valid audience is not
     // specified.
-    private boolean validateAudience(String expectedAudience, List<String> audience) throws OAuthClientAuthnException {
+    private boolean validateAudience(ArrayList<String> expectedAudience, List<String> audience) throws OAuthClientAuthnException {
```

Comment:
Shall we go with more generic collection: i.e. List

Reply:
Fixed